Blackfynn/data-portal
# Dataset page tweaks #1c2qqa

# Description

The purpose of this PR is to add the following tweaks for the dataset page
- Add breadcrumb
- Add markdown for dataset readme 


## Type of change


# How Has This Been Tested?

- Open [a dataset](http://127.0.0.1:5000/browse/#/datasets/16)
- Readme should load and be formatted
- Open Vue devtools
- Inspect `DatasetDetails`
- Add various markdown formatting to the `markdown` data property
- Markdown should be formatted properly
- Scroll to the top
- Use the "View all datasets" breadcrumb link
- Should be taken to the Browse page searching all datasets.

# Checklist:
